MVVM/ViewModel/DriveMode_UIOverlay.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Godot;
+using RoverControlApp.Core;
+using RoverControlApp.MVVM.Model;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace RoverControlApp.MVVM.ViewModel;
+
+public partial class DriveMode_UIOverlay : UIOverlay
+{
+	[Export] 
+	PanelContainer _panelContainer;
+
+	private int _inputMode; 
+
+	public override Dictionary<int, Setting> Presets { get; } = new()
+	{
+		{ 0, new(Colors.DarkGray, Colors.LightGray, "Drive: Compatibility", "Drive: ") },
+		{ 1, new(Colors.LightGreen, Colors.DarkGreen, "Drive: Ackermann","Drive: ") },
+		{ 2, new(Colors.LightSalmon, Colors.DarkRed, "Drive: Crab","Drive: ") },
+		{ 3, new(Colors.Yellow, Colors.Black, "Drive: Spinner","Drive: ") },
+		{ 4, new(Colors.DarkBlue, Colors.LightBlue, "Drive: E-Brake","Drive: ") }
+	};
+
+	public Task KinematicModeChangedSubscriber(MqttClasses.KinematicMode newMode)
+	{
+		ControlMode = (int)newMode;
+
+		return Task.CompletedTask;
+	}
+
+	public Task ControlModeChangedSubscriber(MqttClasses.ControlMode newMode)
+	{
+		_inputMode = (int)newMode;
+		ControlMode = ControlMode;
```

Comment:
found something nice
```suggestion
```

Reply:
Don't remember exactly why I did it but it had something to do with animations. Deleted it. No changes. Seems good to me xd
[d72ca3e](https://github.com/wisniax/RoverControlApp/pull/36/commits/d72ca3ef818e40cc921a178bbdcb8730988ee749)